LogMonitor/src/LogMonitor/EtwMonitor.cpp

```diff
@@ -705,6 +705,69 @@ EtwMonitor::OnRecordEvent(
     return status;
 }
 
+///
+/// Format ETW eventlog into a JSON output
+///
+std::wstring etwJsonFormat(EtwLogEntry* pLogEntry)
+{
+    std::wostringstream oss;

```

Comment:
nit: what does the variable oss mean?

Reply:
Just maintained the old naming, means "out string stream"